Library/Homebrew/cmd/update.rb

```diff
@@ -91,10 +94,23 @@ def update
       end
     end
 
+    # Report changes to Homebrew itself and tapped Git repositories.
+    repo_changes.reject! { |change| change[1] == change[2] }
+    repo_changes.each do |change|
+      tap_name, rev_from, rev_to = change
+      tap_label = if tap_name == "homebrew/homebrew"
+        "Homebrew"
+      else
+        "tap '#{tap_name}'"
+      end
```

Comment:
Thanks for the hint. I was trying to stay below my personal limit of 80 columns (though Homebrew frequently breaks this limit), otherwise I would have used the ternary operator. I'm really not sure what the more readable variant is …